contracts/timed_asset_lock_contract/transaction_builder.py

```diff
@@ -0,0 +1,83 @@
+'''
+This tool is meant to be a CLI help tool that lets you fill in information about the various interactions and
+transactiosn with the timed_asset_lock_contract and to dump the transaction to a base64 string which could then
+be used with tools with non power users
+'''
+
+from algosdk.future import transaction
+from algosdk import encoding
+from akita_inu_asa_utils import get_algod_client, load_developer_config, load_compiled, load_schema
+from .program import compile_app
+
+
+alreadyCollected = {}
+
+
+def collect_info(info_key, store=True):
+    if info_key in alreadyCollected.keys():
+        return alreadyCollected[info_key]
+    value = input("Enter " + info_key + "\n")
+    if store:
+        alreadyCollected[info_key] = value
+    return value
+
+
+def main():
+    config = load_developer_config()
+    client = get_algod_client(config["algodToken"], config["algodAddress"])
+    selection = int(input("Select Transaction To Build...\n"

```

Comment:
I think using argparse could be preferable and makes sure the type of transaction is intentional in the command - it's also useful to make sure args are in the correct form among other features